src/sentry/static/sentry/app/components/pluginConfig.jsx

```diff
@@ -104,7 +108,7 @@ const PluginConfig = React.createClass({
                 </strong>
               </div>
             : null}
-
+          <div dangerouslySetInnerHTML={this.createMarkup()} />
```

Comment:
the way i've dealt with this before is to pass a `key` to the div that will change when this needs to be re-rendered. so, you could either use `this.props.data.doc` as the key or if that's too long/gross do something like have a state variable that is `docKey` and use `componentWillRecieveProps`  (see [here](https://facebook.github.io/react/docs/react-component.html#componentwillreceiveprops)) to change it when the doc changes